index.html

```diff
@@ -1,179 +1,181 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-    <link rel="dns-prefetch" href="//fonts.googleapis.com" />
-    <link
-      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <link rel="stylesheet" href="styles/style.css" />
-
-    <title>Bikes for Refugees</title>
-  </head>
-
-  <body>
-    <div class="header">
-
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+  <link rel="dns-prefetch" href="//fonts.googleapis.com" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,"
+    rel="stylesheet"
+  />
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <link rel="stylesheet" href="styles/style.css" />
+
+  <title>Bikes for Refugees</title>
+</head>
+
+<body>
+  <div class="header">
+
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <div>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </div>
 
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
+      <button>Donate Now</button>
```

Comment:
nice work :100: , can you please change the donate button to an anchor that could be linked to another page?